charts/seed-controlplane/charts/packet-cloud-controller-manager/templates/packet-ccm-deployment.yaml

```diff
@@ -0,0 +1,154 @@
+---
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: packet-cloud-controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: packet-cloud-controller-manager
+    garden.sapcloud.io/role: controlplane
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: packet-cloud-controller-manager
+  template:
+    metadata:
+      labels:
+        app: packet-cloud-controller-manager
+{{- if .Values.podAnnotations }}
+      annotations:
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
+    spec:
+      dnsPolicy: Default
+      hostNetwork: true
+      serviceAccountName: cloud-controller-manager
+      tolerations:
+        # this taint is set by all kubelets running `--cloud-provider=external`
+        # so we should tolerate it to schedule the packet ccm
+        - key: "node.cloudprovider.kubernetes.io/uninitialized"
+          value: "true"
+          effect: "NoSchedule"
+        - key: "CriticalAddonsOnly"
+          operator: "Exists"
+        # cloud controller manager should be able to run on masters
+        - key: "node-role.kubernetes.io/master"
+          effect: NoSchedule
+      containers:
+      - image: {{ index .Values.images "packet-controller-manager" }}
+        name: packet-cloud-controller-manager
+        command:
+          - "./packet-cloud-controller-manager"
+          - "--cloud-provider=packet"
+          - "--leader-elect=false"
+          - "--allow-untagged-cloud=true"
+        resources:
+          requests:
+            cpu: 100m
+            memory: 50Mi
+        env:
+          - name: PACKET_AUTH_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: cloudprovider
+                key: apiToken
+          - name: PACKET_PROJECT_ID
+            valueFrom:
+              secretKeyRef:
+                name: cloudprovider
+                key: projectID
+        {{- if .Values.resources }}
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
+        {{- end }}
+---
+apiVersion: v1
```

Comment:
OK, I added the `--kubeconfig=<path>` line, as well as the volume mounts per the alicloud example, and removed the various permissions (`ServiceAccount`, `ClusterRole`, `ClusterRoleBinding`) for the CCM since they are not necessary.